submissions/bukkyOyetimehin/challenge2- arrays/question2.js

```diff
@@ -0,0 +1,32 @@
+function twoSum(nums, target) {
```

Comment:
Also Excellent. Nice idea to use a Map to store complements you might have encountered before. 
Welldone